### PR TITLE
Add signing step to distroless release

### PR DESCRIPTION
**Pull request by priyawadhwa**
Right now signatures will be stored in gcr.io/distroless-signatures -- this will be switched to the main distroless repo once we are confident signing is working as expected!

### CODE REVIEW

**Comment by priyawadhwa** [comment]
cc @dlorenc

File: cloudbuild.yaml [hunk 1]
```diff
@@ -39,3 +39,10 @@ steps:
   - PROJECT_ID=${PROJECT_ID}
   - COMMIT_SHA=${COMMIT_SHA}
   entrypoint: ./cloudbuild_docker.sh
+- name: gcr.io/projectsigstore/cosign/ci/cosign:v0.2.0
```
Comment (dlorenc) [inline cloudbuild.yaml:42]: Want to grab the digest to be safe?

File: cloudbuild_cosign.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+set -o errexit
+set -o xtrace
+
+
+export KMS_VAL=gcpkms://projects/$PROJECT_ID/locations/global/keyRings/cosign/cryptoKeys/cosign
+# TODO (priyawadhwa@): Store signatures in distroless repo once we are confident this works
+export COSIGN_REPOSITORY=gcr.io/distroless-signatures
+
+
```
Comment (dlorenc) [inline cloudbuild_cosign.sh:11]: For debugging, could you throw a "cosign version" here too?

**Review by dlorenc — COMMENTED** [review]
NICE!!! two nits

File: cloudbuild.yaml (hunk 1 above)
Comment (priyawadhwa) [reply cloudbuild.yaml]: good call, done!

File: cloudbuild_cosign.sh (hunk 2 above)
Comment (priyawadhwa) [reply cloudbuild_cosign.sh:12]: fosh

**Review by dlorenc — APPROVED** [review]

**Pull request merged**